packages/transport/src/modules/accounts/backbone/IdentityClient.ts

```diff
@@ -1,31 +1,11 @@
 import { RESTClient, RESTClientLogDirective } from "../../../core";
 import { ClientResult } from "../../../core/backbone/ClientResult";
-import { IChallengeSignedSerialized } from "../../challenges/data/ChallengeSigned";
-
-export interface PlatformIdentityPostRequest {
-    identityPublicKey: string;
-    devicePassword: string;
-    signedChallenge: IChallengeSignedSerialized;
-    clientId: string;
-    clientSecret: string;
-    identityVersion: number;
-}
-
-export interface PlatformIdentityDevice {
-    id: string;
-    username: string;
-}
-
-export interface PlatformIdentityPostResponse {
-    address: string;
-    device: PlatformIdentityDevice;
-    createdAt: string;
-}
+import { BackboneIdentityPostRequest, BackboneIdentityPostResponse } from "./BackbonePostIdentity";
 
 export class IdentityClient extends RESTClient {
     protected override _logDirective = RESTClientLogDirective.LogResponse;
 
-    public async createIdentity(value: PlatformIdentityPostRequest): Promise<ClientResult<PlatformIdentityPostResponse>> {
-        return await this.post<PlatformIdentityPostResponse>("/api/v1/Identities", value, {});
+    public async createIdentity(value: BackboneIdentityPostRequest): Promise<ClientResult<BackboneIdentityPostResponse>> {
```

Comment:
I think naming is a bit off here

should be `BackbonePostIdentityRequest` and `BackbonePostIdentityResponse`